web/static/js/app.js

```diff
@@ -2,9 +2,12 @@
 
 import React from "react"
 import { render } from "react-dom"
+import { createStore } from "redux"
+import { Provider } from "react-redux"
 
 import RemoteRetro from "./components/remote_retro"
 import RetroChannel from "./services/retro_channel"
+import rootReducer from "./reducers/index"
```

Comment:
i think (think) that you can leave off the `/index` here, as `index.js` is the default file that will be imported if you `import` a directory